Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -647,6 +653,6 @@ def submit(self):
                 workFlow=makeRequest(self.wmagent,d,encodeDict=True)
                 print("...........",n,"submitted")
                 random_sleep()
-            
-
-        
+        if self.testMode and self.countLongWFName>0:
+            print("\n*** FIX NEEDED BEFORE INJECTION: "+str(self.countLongWFName)+" candidate workflows have too long name (>81 characters) ***")
```

Comment:
raise an exception?

Reply:
Fix the text. This is for warning only that the workflows can't be submitted later.

Reply:
Is there a situation where we want to allow those long workflow names? If not, I still think an exception would be better. That way, any PR creating an unnecessarily long workflow name must be fixed.

Reply:
As mentioned in the PR description, if someone would like to use "--wm init" to dump the configuration to run local-test only, we should not break it. This warning is to target for people who will submit relvals later, using "--wm submit, force" after "--wm init".